mokymo-metu-context/src/context/UserContext.jsx

```diff
@@ -30,8 +30,3 @@ const UserProvider = ({ children }) => {
 
 // fourth step - export context and provider
 export { UserContext, UserProvider };
-
-// <UserProvder>
-//  Children
-//     <div></div>
-// </UserProvder>
```

Comment:
galbūt paliekam šitą pavyzdį, kad ateities programuotojai geriau suprastų kaip veikia provideris.

Reply:
man atrodo čia ir taip viskas aišku, galbūt galėtume aprašyti kažkur šitą info, o ne kode palikt.